src/Text/Pandoc/Class.hs

```diff
@@ -477,6 +477,14 @@ liftIOError f u = do
          Left e  -> throwError $ PandocIOError u e
          Right r -> return r
 
+-- | Show potential IO errors to the user continuing execution anyway
+logIOError :: IO a -> PandocIO ()
+logIOError f = do
+  res <- liftIO $ tryIOError f
+  case res of
+    Left e -> report $ IgnoredIOError (E.displayException e)
+    Right _ -> pure ()
```

Comment:
Why not change type to `IO a -> PandocIO a` and change this line to
```
Right x -> return x
```
That makes the function more general.
Either that, or change the type to `IO () -> PandocIO ()`.

Reply:
`IO a -> Pandoc a` would require a default value when the operation fails, so i opted for `IO () -> Pandoc ()`, i amended and rebased